code/obj/machinery/arc_electroplater.dm

```diff
@@ -91,7 +114,7 @@ TYPEINFO(/obj/machinery/arc_electroplater)
 			boutput(user, SPAN_ALERT("You can't plate something this tiny!"))
 			return
 
-		if (W.w_class > src.max_wclass || istype(W, /obj/item/storage/secure))
+		if (W.w_class > src.max_wclass || istype(W, /obj/item/storage/secure) || W.anchored >= ANCHORED)
```

Comment:
```suggestion
		if (W.w_class > src.max_wclass || istype(W, /obj/item/storage/secure) || W.anchored)
```
Unless something has a negative (or fractional, I suppose) value for `anchored`, the check can be simplified (and usually is)